pkgs/by-name/as/asm-lsp/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, stdenv
+, darwin
+}:
+let
+  pname = "asm-lsp";
+  version = "0.4.2";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "bergercookie";
+    repo = "asm-lsp";
+    rev = "v${version}";
+    hash = "sha256-jVWZ0eOivDnpyo6BYef40tnl8roRbfXhA4eZ/Sx0VbU=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs =
+    [
```

Comment:
I'd also move this up a line, for consistency

Reply:
by "up a line do you mean something like this

```nix
 nativeBuildInputs = [
    pkg-config
  ];
  buildInputs =
    [
      openssl
    ]
    ++ lib.optionals stdenv.isDarwin [
      darwin.apple_sdk.frameworks.Security
    ];
```